parsl/dataflow/rundirs.py

```diff
@@ -22,7 +22,7 @@ def make_rundir(config=None, path=None):
     try:
         if not path:
             path = None
-        elif config["globals"].get('runDir', None):
+        elif config.get("globals", {}).get('runDir'):
```

Comment:
:+1:  Makes sense, at the point where rundirs are created, we haven't set global defaults yet.